packages/frontend/src/plugin.ts

```diff
@@ -205,13 +205,13 @@ type HandlerDef = {
 		handler: (note: Misskey.entities.Note) => void;
 	};
 	note_view_interruptor: {
-		handler: (note: Misskey.entities.Note) => unknown;
+		handler: (note: Misskey.entities.Note) => Misskey.entities.Note;
```

Comment:
```suggestion
		handler: (note: Misskey.entities.Note) => Misskey.entities.Note | null;
```

nullが返ると非表示になる仕様がある https://github.com/misskey-dev/misskey/pull/12135